src/shared/components/Animations/ListItem.tsx

```diff
@@ -24,13 +25,23 @@ const ListItem: React.FunctionComponent<ListItemProps> = ({
   action,
   preview,
   popover,
+  onAllClick,
```

Comment:
naming: I have no Idea what 'onAllClick' means. Is there a link labeled 'All'? or is it an event that will be launched on all links?

Reply:
I decided to swap it with the other. so now there's `onClick` and `onTitleClick`